### PR TITLE
support postfix conditionals

### PR DESCRIPTION
**Pull request by aiomaster**
Try to fix inline conditions:
https://github.com/trailblazer/cells-erb/issues/5
https://github.com/apotonick/erbse/issues/4
https://github.com/apotonick/erbse/issues/3

### CODE REVIEW

**Comment by apotonick** [comment]
Have you tested it in your app? :thinking: :grimacing:

File: lib/erbse/parser.rb [hunk 1]
```diff
@@ -4,6 +4,7 @@ class Parser
     ERB_EXPR = /<%(=+|-|\#|@\s|%)?(.*?)[-=]?%>(\n)*/m # this is for backward-compatibility.
     # BLOCK_EXPR     = /\s*((\s+|\))do|\{)(\s*\|[^|]*\|)?\s*\Z/
     BLOCK_EXPR = /\b(if|unless)\b|\sdo\s*$|\sdo\s+\|/
```
Comment (apotonick) [inline lib/erbse/parser.rb]: So you simply removed the `if|unless` pattern? And it still works? Ingenious!

Comment (aiomaster) [reply lib/erbse/parser.rb]: Hi @apotonick,
maybe you've overseen it. I removed the `if|unless` pattern only in the `BLOCK_EXPR_OUT` constant that is used only in expression tags `<%= ... %>`. So there the keywords `if` and `else` should be interpreted as inline or postfix condition.
But in execution tags `<% ... %>` I want to have the old behaviour and interprete `if|unless` as block expressions. So I use the old constant `BLOCK_EXPR` there with `if|unless`.

The only problem I see with this solution is this one:
```
<% foo = 'bar' if true %>
<%= foo %>
```
So maybe I find another way of solving this.

Comment (apotonick) [reply lib/erbse/parser.rb]: Can you add a breaking test for this, please? We can then discuss the solution.

**Comment by aiomaster** [comment]
Now I add support for postfix conditions in execution tags also, so the following should work:
```
<%= 'test' if true %>
<% foo = 'bar' if true %>
<%= foo %>
<% if true %>
  blah
<% end %>
```
and it should output:
```
test
bar
blah
```

**Comment by Fodoj** [comment]
Any progress here?

**Comment by apotonick** [comment]
0.1.2 is released.

**Pull request merged**